crates/pathfinder/src/rpc/v02/method/chain_id.rs

```diff
@@ -0,0 +1,36 @@
+use crate::rpc::v02::RpcContext;
+
+crate::rpc::error::generate_rpc_error_subset!(ChainIdError);
+
+#[allow(dead_code)]
+pub async fn chain_id(context: std::sync::Arc<RpcContext>) -> Result<String, ChainIdError> {
```

Comment:
Should we change the methods to take `AsRef<RpcContext>` instead? Then we don't have to worry about the `RpcContext::for_tests` + `Arc` issue.

```suggestion
pub async fn chain_id(context: &RpcContext) -> Result<String, ChainIdError> {
```